packages/rebber-plugins/__tests__/fixtures/mix-7.fixture.md

```diff
@@ -2,8 +2,8 @@
 | cell | `code                |
 |      | with line break`     |
 +------+----------------------+
-|  ```text                    |
-| line 1                      |
-| line2                       |
+|```text                      |
+|line 1                       |
+|line2                        |
```

Comment:
I would like to keep this test as is, so updating the fixture is not the way to go. You should however update the snapshot, which requires reverting changes on this file and running `npm test -- packages/rabber-plugins -u` at the root of the repository.

Reply:
Since I see you asked if you should add tests: you can, but I'm not requiring it. I know which tests I'd like to have (ie. micromark compatibility), so I'll add a few if you don't after merging.

Reply:
Thanks for the comments, coming back to this now, will do as you asked